src/ScreenWidget.py

```diff
@@ -1,160 +1,94 @@
 #!/usr/bin/python
-
-from PyQt5.QtWidgets import QWidget, QLabel, QVBoxLayout, QHBoxLayout, QPushButton, QMenu
-from PyQt5.QtGui import QPainter, QPixmap, QImage, QColor, qRgb, qRgba
-from PyQt5.QtCore import Qt, pyqtSignal, QRect
-
-import numpy as np
+import functools
 import cv2
+from PyQt5.QtWidgets import QWidget, QVBoxLayout, QAction
+from PyQt5.QtCore import pyqtSignal
+from PyQt5 import QtCore
+import pyqtgraph as pg
 
 from Helper import *
 
-WIDTH_SCREEN = WS =  500
-HEIGHT_SCREEN = HS = 375
-
-CONVERSION_PX = WIDTH_FRAME / WIDTH_SCREEN
-
 
-class ScreenWidget(QLabel):
+class ScreenWidget(pg.GraphicsView):
 
     selected = pyqtSignal(int, int)
     cleared = pyqtSignal()
 
     def __init__(self, filename=None, model=None, parent=None):
-        QWidget.__init__(self, parent=parent)
+        super().__init__(parent=parent)
```

Comment:
Can the screenwidget initialize in the same aspect ratio as our frames (3000x4000)? currently it looks like they're maybe 640 x 240 (half VGA) or so? that way there would be no black border at full-frame.